cmd/clip.go

```diff
@@ -0,0 +1,50 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/atotto/clipboard"
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// clipCmd represents the clip command
+var clipCmd = &cobra.Command{
```

Comment:
This feature is not related to multi-folder support, right? If so, could you open another PR for this feature?